guidance/models/_grammarless.py

```diff
@@ -104,8 +104,14 @@ def __init__(self, tokenizer):
         else:
             raise Exception("The tokenizer given was not of a recognized type!")
 
+        self._orig_tokenizer = tokenizer
+
         super().__init__(byte_tokens, bos_token_id, eos_token_id)
 
+    def __call__(self, byte_string):
+        """Returns a list of tokens that represent the given byte string."""
+        return self._orig_tokenizer.encode(byte_string)
```

Comment:
I suspect that this is an overly simple implementation. I hadn't realised that it wasn't already implemented.